libs/sdk-ui-ext/src/internal/constants/dropdowns.ts

```diff
@@ -149,6 +149,12 @@ export const calculationDropdownItems: IDropdownItem[] = [
         info: CalculateAs.DIFFERENCE,
         icon: "gd-dropdown-icon-calculated-as-difference",
     },
+    {
+        title: comparisonMessages.calculatedAsChangeDifference.id,
+        value: CalculateAs.CHANGE_DIFFERENCE,
+        info: CalculateAs.CHANGE_DIFFERENCE,
+        icon: "gd-dropdown-icon-calculated-as-change-difference",
+    },
```

Comment:
These items should be in order Change -> Change (Difference) -> Ratio -> Difference

Reply:
Fixed, thank you!